.circleci/config.yml

```diff
@@ -7,6 +7,7 @@ jobs:
           LOG_LEVEL: DEBUG
     steps:
       - checkout
+      - run: sudo apt-get update && sudo apt-get install -y cmake # For installing snappy
```

Comment:
Is this line relevant to the multiple address support?

Reply:
This line was needed to run the tests. I've removed it because the current master branch includes the change.